cicd/gitops/monitoring-argo-cd-instances.adoc

```diff
@@ -0,0 +1,15 @@
+:_content-type: ASSEMBLY
+[id="monitoring-argo-cd-instances"]
+= Monitoring Argo CD instances
+include::_attributes/common-attributes.adoc[]
+:context: monitoring-argo-cd-instances
+
+toc::[]
+
+By deffault, the {gitops-title} automatically detects a newly installed Argo CD instance in the `openshift-gitops` namespace and wires them up with the cluster's monitoring stack with an alert for reporting out-of-sync application.
```

Comment:
also the openshift-gitops namespace isn't a requirement, so maybe it would be even better to say "automatically detects an installed Argo CD instance in your defined namespace...etc" (this would also mean changing the rest of the doc accordingly to make it suit this logic of using a defined namespace)